### ISSUE TITLE
Incorrect web interface operation

### ISSUE THREAD

**Issue by zorro-one** [RaidMax/IW4M-Admin#79]
After upgrading to Version 2.3 Pre-Release 8 Revision 0 on the web interface clicking on the "**Penalties**", "**Admins**" links and the **player name** leads to the error "**Sorry! An error occurred while processing your request**". This makes it impossible to ban a player. Without errors displays **Home** and **Top Players**.
![Remote_control](https://user-images.githubusercontent.com/53860363/62710267-aeaec880-b9ff-11e9-94b2-dfc42bff5ab1.png)

The IW4MAdmin console displays the following:

`[08.07.2019 20:58:31.308] - Error: [Webfront] One or more compilation failures occurred:
c1tum1tz.zdm(23,107): error CS0234: The type or namespace name 'Objects' does not exist in the namespace 'SharedLibraryCore' (are you missing an assembly reference?)
c1tum1tz.zdm(450,89): error CS0234: The type or namespace name 'Objects' does not exist in the namespace 'SharedLibraryCore' (are you missing an assembly reference?)
D:\ZORRO\IW4MAdmin-2.2-stable\Views\Penalty\List.cshtml(9,81): error CS0234: The type or namespace name 'Objects' does not exist in the namespace 'SharedLibraryCore' (are you missing an assembly reference?)`

Along with this, there is no way to login from a remote machine (from a local machine one without problems).

**Comment by RaidMax** [comment]
Hey, looks like you upgraded as you were supposed to. Glad you actually read the readme.
There was another change from 2.2 - pre release which uses compiled views for the webfront instead of raw html. If you delete the `View` folder in your IW4MAdmin folder, everything should be fine!

**Comment by zorro-one** [comment]
> If you delete the `Views` folder in your IW4MAdmin folder, everything should be fine!

Thank! And this problem is solved!

The question remains open:
> Along with this, there is no way to login from a remote machine (from a local machine one without problems).

**Comment by RaidMax** [comment]
You would need to use !setpassword or !requesttoken ingame to set the password or get the one use token to login.

!requesttoken is more secure and I recommend. It will give you the code and your client ID

**Comment by zorro-one** [comment]
Trying to get a token using the web interface console on the local machine led me to bewilderment:

Query `!requesttoken` displayed `client ID 1`
Query `!find zorro` displayed `client ID 10`
![IW4MAdmin_03](https://user-images.githubusercontent.com/53860363/62763621-b53b4f80-ba94-11e9-9ad6-166757a6ed83.png)


Well, ok. I entered `client ID 1`, I entered the received `token` and the page became unavailable by domain name.
Ok. I entered the ip-address and the webinterface booted, but without authorization...

Now I am afraid to enter `client ID` and `token` for login:
![IW4MAdmin_04](https://user-images.githubusercontent.com/53860363/62764003-8c678a00-ba95-11e9-8614-f415971f45f1.png)


Is this a bug? Or am I doing something wrong?
